docs/source/markdown/podman-create.1.md

```diff
@@ -1000,6 +1000,14 @@ can override the working directory by using the **-w** option.
 $ podman create alpine ls
 ```
 
+### Run a container with just the SETUID and SETGID Capabilities
```

Comment:
Suggest rename to "Set default capabilities for container to SETUID and SETGID" or similar

Reply:
Removed these commands, since they don't make sense for podman run/create any longer.